doc/sources/samples.rst

```diff
@@ -65,11 +65,11 @@ If you want to run them locally, refer to `these instructions
 .. toctree::
    :maxdepth: 1
 
-   Intel® Extension for Scikit-Learn* Getting Started Sample <https://github.com/oneapi-src/oneAPI-samples/tree/master/AI-and-Analytics/Getting-Started-Samples/Intel_Extension_For_SKLearn_GettingStarted>
-   Intel® Extension for Scikit-Learn*: SVC for Adult dataset Performance Sample <https://github.com/oneapi-src/oneAPI-samples/tree/master/AI-and-Analytics/Features-and-Functionality/Intel_Extension_For_SKLearn_Performance_SVC_Adult>
+   Intel® Extension for Scikit-Learn* Getting Started Sample <https://github.com/uxlfoundation/oneAPI-samples/tree/master/AI-and-Analytics/Getting-Started-Samples/Intel_Extension_For_SKLearn_GettingStarted>
+   Intel® Extension for Scikit-Learn*: SVC for Adult dataset Performance Sample <https://github.com/uxlfoundation/oneAPI-samples/tree/master/AI-and-Analytics/Features-and-Functionality/Intel_Extension_For_SKLearn_Performance_SVC_Adult>
```

Comment:
`oneAPI-samples` are currently in `oneapi-src`. It makes sense to leave the links as they are, at least for now.

Reply:
Thanks, updated both of them.